mypy.ini

```diff
@@ -11,7 +11,8 @@ exclude = (?x)(
 	| ^.tox/
 	| ^.egg/
 	| ^pkg_resources/tests/data/my-test-package-source/setup.py$ # Duplicate module name
-	| ^.+?/(_vendor|extern)/ # Vendored
+	| ^pkg_resources/_vendor/ # Vendored
+	| ^setuptools/_vendor/ # Vendored
```

Comment:
Why not?
```suggestion
	| ^.+?/_vendor/ # Vendored
```
or even
```suggestion
	| /_vendor/ # Vendored
```
I think it's safe to assume any `_vendor` folder will be vendored. (unless you just wanted to be extra explicit between pkg_resources and setuptools)